src/Microsoft.ML.StandardLearners/Standard/Online/OnlineLearnerStatic.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using System;
+
+namespace Microsoft.ML.Trainers
```

Comment:
>Microsoft.ML.Trainers [](start = 10, length = 21)

change namespace

Reply:
Actually moving to ML.StaticPipe namespace won't fix it by itself. All the contexts are in Microsoft.ML. 

@Tomfinley move the contexts in StaticPipe as well?

---
In reply to: [221111449](https://github.com/dotnet/machinelearning/pull/1084#discussion_r221111449) [](ancestors = 221111449)

Reply:
Yes

---
In reply to: [221299214](https://github.com/dotnet/machinelearning/pull/1084#discussion_r221299214) [](ancestors = 221299214,221111449)

Reply:
No. If the user opts into the static pipe, they will see the trainers. It's good enough for me

---
In reply to: [221299214](https://github.com/dotnet/machinelearning/pull/1084#discussion_r221299214) [](ancestors = 221299214,221111449)

Reply:
The context is in Microsoft.ML, i don't see anything implicitely import staticPipe.

---
In reply to: [221307154](https://github.com/dotnet/machinelearning/pull/1084#discussion_r221307154) [](ancestors = 221307154,221299214,221111449)